README.md

```diff
@@ -1518,6 +1518,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Triman Kaur](https://github.com/Trimankaur)
   - [Tung Pham](https://github.com/TungPhamQ)
   - [Tushar Pathak](https://github.com/tushar54594)
+  - [TUshar Agarwal](https://github.com/TusharAgarwal18)
```

Comment:
Your Name should be in a proper format. The 'U' is capital ...please make it small

Reply:
Thanks for pointing it out.